### ISSUE TITLE
BCC/BTC in Bitfinex

### ISSUE THREAD

**Issue by nmiculinic** [ccxt/ccxt#494]
Bitfinex ticker:
```
             'ticker': {'ask': 1.0,
                        'average': 0.9766,
                        'baseVolume': 0.02833264,
                        'bid': 0.9532,
                        'change': None,
                        'close': None,
                        'datetime': '2017-11-08T10:48:26.000Z',
                        'first': None,
                        'high': 0.9888,
                        'info': {'ask': '1.0',
                                 'bid': '0.9532',
                                 'high': '0.9888',
                                 'last_price': '0.9532',
                                 'low': '0.953',
                                 'mid': '0.9766',
                                 'timestamp': '1510138105.8681781',
                                 'volume': '0.02833264'},
                        'last': 0.9532,
                        'low': 0.953,
                        'open': None,
                        'percentage': None,
                        'quoteVolume': None,
                        'symbol': 'BCC/BTC',
                        'timestamp': 1510138105868.1782,
                        'vwap': None}}}
```
```
             'info': {'base': 'BCC',
                      'baseId': 'BCC',
                      'id': 'BCCBTC',
                      'info': {'expiration': 'NA',
                               'initial_margin': '30.0',
                               'maximum_order_size': '2000.0',
                               'minimum_margin': '15.0',
                               'minimum_order_size': '0.004',
                               'pair': 'bccbtc',
                               'price_precision': 5},
                      'limits': {},
                      'precision': {'price': 5},
                      'quote': 'BTC',
                      'quoteId': 'BTC',
                      'symbol': 'BCC/BTC'},
```

In Bitfinex it's something called CST Core (I guess, I see it in their wallets), but I cannot find this pair in Bitfinex trades. Also BCC/BTC is usually Bitconnect/Bitcoin and it's confusing what's happening with this pair.

**Comment by kroitor** [comment]
Bitfinex has both BCH and BCC.
BCH = Bitcoin Cash 
BCC = Bitconnect 
The above corresponds to coinmarketcap, and should be true throughout the ccxt library.

This table might shed some light:

```
mbp:ccxt igorkroitor$ node examples/js/symbols bitfinex
Instantiating bitfinex exchange (printSymbols @ symbols.js:24)
bitfinex has 60 symbols: AVT/BTC, AVT/ETH, AVT/USD, BCC/BTC, BCC/USD, BCH/BTC, BCH/ETH, BCH/USD, BCU/BTC, BCU/USD, BT1/BTC, BT1/USD, BT2/BTC, BT2/USD, BTC/USD, BTG/BTC, BTG/USD, DASH/BTC, DASH/USD, DAT/BTC, DAT/ETH, DAT/USD, EDO/BTC, EDO/ETH, EDO/USD, EOS/BTC, EOS/ETH, EOS/USD, ETC/BTC, ETC/USD, ETH/BTC, ETH/USD, ETP/BTC, ETP/ETH, ETP/USD, IOT/BTC, IOT/ETH, IOT/USD, LTC/BTC, LTC/USD, NEO/BTC, NEO/ETH, NEO/USD, OMG/BTC, OMG/ETH, OMG/USD, QTUM/BTC, QTUM/ETH, QTUM/USD, RRT/BTC, RRT/USD, SAN/BTC, SAN/ETH, SAN/USD, XMR/BTC, XMR/USD, XRP/BTC, XRP/USD, ZEC/BTC, ZEC/USD (printSymbols @ symbols.js:37)
id     | symbol   | base | quote | baseId | quoteId | info            | precision      
---------------------------------------------------------------------------------------
BCCBTC | BCC/BTC  | BCC  | BTC   | BCC    | BTC     | [object Object] | [object Object]
BCCUSD | BCC/USD  | BCC  | USD   | BCC    | USD     | [object Object] | [object Object]
BCHBTC | BCH/BTC  | BCH  | BTC   | BCH    | BTC     | [object Object] | [object Object]
BCHETH | BCH/ETH  | BCH  | ETH   | BCH    | ETH     | [object Object] | [object Object]
BCHUSD | BCH/USD  | BCH  | USD   | BCH    | USD     | [object Object] | [object Object]
BCUBTC | BCU/BTC  | BCU  | BTC   | BCU    | BTC     | [object Object] | [object Object]
BCUUSD | BCU/USD  | BCU  | USD   | BCU    | USD     | [object Object] | [object Object]
BT1BTC | BT1/BTC  | BT1  | BTC   | BT1    | BTC     | [object Object] | [object Object]
BT1USD | BT1/USD  | BT1  | USD   | BT1    | USD     | [object Object] | [object Object]
BT2BTC | BT2/BTC  | BT2  | BTC   | BT2    | BTC     | [object Object] | [object Object]
BT2USD | BT2/USD  | BT2  | USD   | BT2    | USD     | [object Object] | [object Object]
BTCUSD | BTC/USD  | BTC  | USD   | BTC    | USD     | [object Object] | [object Object]
BTGBTC | BTG/BTC  | BTG  | BTC   | BTG    | BTC     | [object Object] | [object Object]
BTGUSD | BTG/USD  | BTG  | USD   | BTG    | USD     | [object Object] | [object Object]
```

This link is also worth it: https://www.bitfinex.com/posts/213/review

So, answering your question, I don't see any problem with BCC on Bitfinex inside ccxt, do you?

**Comment by nmiculinic** [comment]
Yes, I do.

As per coinmarketcap, the price of BCC is around 0.032 BTC. Not ~1BTC as returned from the API. Thus there has been some mixup between pairs.

**Comment by kroitor** [comment]
@nmiculinic got it, I am looking at why this might be happening.

**Comment by kroitor** [comment]
Hm...
https://www.bitfinex.com/legal/cst
https://www.bitfinex.com/legal/cst/bitcoin_unlimited

That's a little confusing indeed. Any ideas on what we should do about it?

**Comment by nmiculinic** [comment]
That's added complexity I'd rather avoid if not necessary. Let's say the de facto standard is coinmarketcap --> BCC is Bitconnect. 

For this specific Bitfinex stuff...let's call them CST_BCC for example? 

How does that sound to you?

**Comment by kroitor** [comment]
@nmiculinic sounds good, I'll do that right away, hold on plz! )

**Comment by kroitor** [comment]
This was fixed in 1.10.23. Thanks for reporting!
